dozer/cogs/moderation.py

```diff
@@ -292,88 +298,133 @@ async def on_message(self, message):
-            if 1024 > len(before.content) > 0:
-                e.add_field(name="Old message", value=before.content)
-            elif len(before.content) != 0:
-                e.add_field(name="Old message", value=before.content[0:1023])
-                e.add_field(name="Old message continued", value=before.content[1024:2000])
-            elif len(before.content) == 0 and before.edited_at is not None:
-                for i in before.embeds:
-                    e.add_field(name="Title", value=i.title)
-                    e.add_field(name="Description", value=i.description)
-                    e.add_field(name="Timestamp", value=i.timestamp)
-                    for x in i.fields:
-                        e.add_field(name=x.name, value=x.value)
-                    e.add_field(name="Footer", value=i.footer)
-            if before.attachments:
-                e.add_field(name="Attachments", value=", ".join([i.url for i in before.attachments]))
-            if 0 < len(after.content) < 1024:
-                e.add_field(name="New message", value=after.content)
-            elif len(after.content) != 0:
-                e.add_field(name="New message", value=after.content[0:1023])
-                e.add_field(name="New message continued", value=after.content[1024:2000])
-            elif len(after.content) == 0 and after.edited_at is not None:
-                for i in after.embeds:
-                    e.add_field(name="Title", value=i.title)
-                    e.add_field(name="Description", value=i.description)
-                    e.add_field(name="Timestamp", value=i.timestamp)
-                    for x in i.fields:
-                        e.add_field(name=x.name, value=x.value)
+            guild_id = str(before.guild.id)
+            channel_id = str(before.channel.id)
+            user_id = str(before.author.id)
+            message_id = str(before.id)
+            link = f"https://discordapp.com/channels/{str(guild_id)}/{str(channel_id)}/{str(message_id)}"
+            embed = discord.Embed(title="Message Edited",
+                                  description=f"[MESSAGE]({link}) From {before.author.mention}"
+                                              f"\nEdited In: {str(before.channel.mention)}", color=0xFFC400,
+                                  timestamp=after.edited_at)
+            embed.set_thumbnail(url=after.author.avatar_url)
+            if before.content:
+                embed.add_field(name="Original", value=str(before.content[0:1023]), inline=False)
+                if len(before.content) > 1024:
+                    embed.add_field(name="Original Continued", value=str(before.content[1024:2000]), inline=False)
+                embed.add_field(name="Edited", value=str(after.content[0:1023]), inline=False)
+                if len(after.content) > 1024:
+                    embed.add_field(name="Edited Continued", value=str(after.content[1024:2000]), inline=False)
```

Comment:
Same as  my comments on L366-L382, these `str(...)` calls look like they could be removed.